app/src/routes/v1/permission/objectPermission.js

```diff
@@ -0,0 +1,32 @@
+const routes = require('express').Router();
```

Comment:
might want to name this variable `router` like you did for the other route files (?)

Reply:
I'll fix that on the next PR.